sdk/appcenter-auth/src/main/java/com/microsoft/appcenter/auth/Auth.java

```diff
@@ -74,6 +76,32 @@
  */
 public class Auth extends AbstractAppCenterService implements NetworkStateHelper.Listener {
 
+    @VisibleForTesting
+    static final ILoggerCallback AUTHENTICATION_EXTERNAL_LOGGER = new ILoggerCallback() {
+
+        @Override
+        public void log(String tag, Logger.LogLevel logLevel, String message, boolean containsPII) {
+            if (!containsPII) {
+                String prefixedTag = LOG_TAG + TAG_DELIMITER + tag;
+                if (Logger.LogLevel.VERBOSE == logLevel) {
+                    AppCenterLog.verbose(prefixedTag, message);
+                } else if (Logger.LogLevel.INFO == logLevel) {
+                    AppCenterLog.info(prefixedTag, message);
+                } else if (Logger.LogLevel.WARNING == logLevel) {
+                    AppCenterLog.warn(prefixedTag, message);
+                } else if (Logger.LogLevel.ERROR == logLevel) {
+                    AppCenterLog.error(prefixedTag, message);
+                }
+            }
+        }
+    };
+
+    /**
+     * Delimiter between two tags.
+     */
+    @VisibleForTesting
+    static final String TAG_DELIMITER = ":";
```

Comment:
constant strings are usually at the top of classes

Reply:
Seems this is a correct location. It is already at the top.